.gitignore

```diff
@@ -35,3 +35,5 @@ out/
 
 ### VS Code ###
 .vscode/
+
+.env
```

Comment:
It would be good if you put a comment so we know what type is being ignored in gitignore, such as `### VS Code ###`